bin/prettier.js

```diff
@@ -232,6 +232,19 @@ if (stdin) {
 
     let output;
 
+    if (argv["list-different"]) {
```

Comment:
I would put this just above `const start = Date.now();` since we don't really need those two variables to do that block of code.